.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, edited, synchronize, reopened]
+  push:
+    branches:
+      - "main"
+jobs:
+  CI:
+    runs-on: ubuntu-latest

```

Comment:
Should we use our hosted-runners?

Reply:
I think self hosted runners should be used on a per need basis, not as default.
We do not pay for GH hosted runners on our open source repo, so here we would only add cost without any gain